resteasy-client-microprofile/src/test/java/org/jboss/resteasy/microprofile/client/ClientHeadersFactoryCDITest.java

```diff
@@ -36,24 +36,26 @@
 
       @Inject
       @RestClient
-      private TestResourceIntf service;
+      private SubClassResourceIntf service;
 
       public String work() {
          return service.hello("Stefano");
       }
    }
 
    @Path("/")
```

Comment:
It would be better if the verification that incoming headers are propagated from parent interface methods is added as a separate test.

This changes removes the verification that passing client headers is inherited into a child interface

Reply:
Sorry, ignore this. Not sure why I thought the subclass was already there